main.js

```diff
@@ -150,7 +150,8 @@ let conn; // connection object from mcproxy for the client variable
 options = {
 	host: config.get("minecraftserver.hostname"),
 	port: config.get("minecraftserver.port"),
-	version: config.get("minecraftserver.version")
+	version: config.get("minecraftserver.version"),
+	favicon: config.get("minecraftserver.favicon")
```

Comment:
favicon should prob not be stored in the mineflayer options, mabye store it in its own variable just for consistency